CHANGES.md

```diff
@@ -8,8 +8,14 @@ To be released.
 
 ### Deprecated APIs
 
+ -  Removed `BlockLocator(Func<long, BlockHash?>, Func<BlockHash, long>, int)`
```

Comment:
I think that we need to move this sentence to the Backward-incompatible API changes section.